xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2794,6 +2808,8 @@ void CVideoPlayer::HandleMessages()
             mode.trickplay = true;
             mode.sync = true;
             m_messenger.Put(new CDVDMsgPlayerSeek(mode));
+            UpdateContentState();
```

Comment:
this is not correct. content reflects the streams we opened in OpenStream. VP does not change streams on seek.

Reply:
sorry, diff confused me. this is not seek here.